core/shared/src/main/scala/io.buildo.metarpheus/core/extractors/controller.scala

```diff
@@ -12,10 +12,16 @@ package object controller {
       case Mod.Annot(Ctor.Ref.Name("command")) => "post"
     }.get
 
+  private[this] def isAuthParam(param: scala.meta.Term.Param): Boolean =
+    param.decltpe match {
```

Comment:
This can be shortened to

```scala
param.decltpe.exists(_.isEqual(Type.Name("Auth"))
```

Note: you have to import `scala.meta.contrib._` for having `isEqual` on trees.

Reply:
if you go with this solution, I would also declare `Type.Name("Auth")` once in a private value (instead of doing that inline)